packages/terra-consumer-nav/src/components/modal/Modal.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../variables';
+
+:local {
+  .modal {
+    background: var(--terra-consumer--modal-background, #fff);
+    border-radius: 5px;
+    display: block;
+    left: 20px;
+    overflow: auto;
+    position: fixed;
+    right: 20px;
+    top: 60px;
+
+    @media screen and (max-width: $terra-tiny-breakpoint) {
+      left: 10px;
+      right: 10px;
+    }
+  }
+
+  .modal-header {
+    display: inline-block;
+    left: 0;
+    position: fixed;
+    right: 0;
+    text-align: center;
+    top: 15px;
+  }
+
+  .modal-title {
+    color: var(--terra-consumer--modal-title-color, #fff);
+    font-size: 20px;
+    line-height: 30px;
+    text-align: center;
+  }
+  
+  .overlay {
+    background-color: var(--terra-consumer--overlay-background, #64696c);
```

Comment:
Discussed, @Mahmud-Khan will be logging a issue to terra to do this, and leave a comment in the code here pointing to that issue. We will pick that up when we have time